platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskInCalendar.java

```diff
@@ -0,0 +1,96 @@
+package org.exoplatform.platform.qa.ui.platform.task;
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.commands.Click;
+import org.exoplatform.platform.qa.ui.calendar.pageobject.CalendarHomePage;
+import org.exoplatform.platform.qa.ui.calendar.pageobject.CalendarManagement;
+import org.exoplatform.platform.qa.ui.calendar.pageobject.TaskManagement;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.core.PLFData;
+import org.exoplatform.platform.qa.ui.core.context.BugInPLF;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceHomePage;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceManagement;
+import org.exoplatform.platform.qa.ui.task.pageobject.ProjectsManagement;
+import org.exoplatform.platform.qa.ui.task.pageobject.TasksManagement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selectors.byText;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.taskmanagement.TaskManagementLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+/**
+ * Created by esmaoui on 02/03/2018.
+ */
+@Tag("sniff")
+@Tag("task")
+public class TaskInCalendar extends Base {
+
+    HomePagePlatform homePagePlatform;
+
+    SpaceManagement  spaceManagement;
+
+    SpaceHomePage    spaceHomePage;
+
+    TasksManagement  tasksManagement;
+    TaskManagement   taskManagement;
+
+    ProjectsManagement projectsManagement;
+    CalendarManagement calendarManagement;
+    CalendarHomePage   calendarHomePage;
+
+
+    ManageLogInOut manageLogInOut;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+
+        homePagePlatform = new HomePagePlatform(this);
+        spaceHomePage = new SpaceHomePage(this);
+        spaceManagement = new SpaceManagement(this);
+        tasksManagement = new TasksManagement(this);
+        projectsManagement= new ProjectsManagement(this);
+        calendarManagement = new CalendarManagement(this);
+        calendarHomePage = new CalendarHomePage(this);
+        manageLogInOut = new ManageLogInOut(this);
+        manageLogInOut.signInCas(PLFData.USER_ROOT, PLFData.password);
+
+    }
+    /*
+    bug TA-605
+   */
+
+    @Test
+    @BugInPLF("TA-605")
```

Comment:
To be removed since the issue is set as Resolved.

Reply:
corrected